assets/models/standard_v2/config.min.json

```diff
@@ -0,0 +1 @@
: 0.91, "autohotkey": 0.5, "autoit": 0.5, "awk": 0.5, "batch": 0.94, "bazel": 0.5, "bib": 0.95, "bmp": 0.95, "bzip": 0.95, "c": 0.94, "cab": 0.95, "cat": 0.95, "chm": 0.95, "clojure": 0.95, "cmake": 0.95, "cobol": 0.5, "coff": 0.95, "coffeescript": 0.65, "cpp": 0.94, "crt": 0.5, "crx": 0.95, "cs": 0.95, "csproj": 0.95, "css": 0.91, "csv": 0.9, "dart": 0.95, "deb": 0.95, "dex": 0.95, "dicom": 0.95, "diff": 0.95, "dm": 0.95, "dmg": 0.95, "doc": 0.95, "dockerfile": 0.95, "docx": 0.95, "dsstore": 0.95, "dwg": 0.95, "dxf": 0.95, "elf": 0.95, "elixir": 0.95, "emf": 0.95, "eml": 0.95, "epub": 0.95, "erb": 0.95, "erlang": 0.95, "flac": 0.95, "flv": 0.95, "fortran": 0.95, "gemfile": 0.95, "gemspec": 0.95, "gif": 0.95, "gitattributes": 0.95, "gitmodules": 0.95, "go": 0.95, "gradle": 0.93, "groovy": 0.9, "gzip": 0.95, "h5": 0.95, "handlebars": 0.95, "haskell": 0.95, "hcl": 0.95, "hlp": 0.95, "htaccess": 0.95, "html": 0.7, "icns": 0.95, "ico": 0.95, "ics": 0.95, "ignorefile": 0.95, "ini": 0.85, "internetshortcut": 0.95, "ipynb": 0.95, "iso": 0.95, "jar": 0.85, "java": 0.95, "javabytecode": 0.95, "javascript": 0.9, "jinja": 0.95, "jp2": 0.95, "jpeg": 0.95, "json": 0.85, "jsonl": 0.91, "julia": 0.94, "kotlin": 0.85, "latex": 0.95, "lha": 0.95, "lisp": 0.9, "lnk": 0.95, "lua": 0.95, "m3u": 0.95, "m4": 0.75, "macho": 0.95, "makefile": 0.95, "markdown": 0.94, "matlab": 0.95, "mht": 0.95, "midi": 0.95, "mkv": 0.95, "mp3": 0.95, "mp4": 0.95, "mscompress": 0.95, "msi": 0.95, "mum": 0.95, "npy": 0.85, "npz": 0.95, "nupkg": 0.95, "objectivec": 0.92, "ocaml": 0.95, "odp": 0.95, "ods": 0.95, "odt": 0.95, "ogg": 0.95, "one": 0.95, "onnx": 0.8, "otf": 0.95, "outlook": 0.95, "parquet": 0.95, "pascal": 0.95, "pcap": 0.95, "pdb": 0.5, "pdf": 0.95, "pebin": 0.95, "pem": 0.95, "perl": 0.95, "php": 0.95, "pickle": 0.95, "png": 0.95, "po": 0.95, "postscript": 0.95, "powershell": 0.6, "ppt": 0.5, "pptx": 0.95, "prolog": 0.95, "proteindb": 0.95, "proto": 0.95, "psd": 0.95, "python": 0.95, "pythonbytecode": 0.95, "qt": 0.95, "r": 0.95, "rar": 0.95, "rdf": 0.95, "rpm": 0.95, "rst": 0.95, "rtf": 0.95, "ruby": 0.9, "rust": 0.95, "scala": 0.94, "scss": 0.95, "sevenzip": 0.95, "sgml": 0.95, "shell": 0.95, "smali": 0.95, "snap": 0.5, "solidity": 0.95, "sql": 0.9, "sqlite": 0.95, "squashfs": 0.95, "srt": 0.95, "stlbinary": 0.55, "stltext": 0.91, "sum": 0.95, "svg": 0.95, "swf": 0.95, "swift": 0.95, "tar": 0.95, "tcl": 0.95, "textproto": 0.95, "tga": 0.95, "thumbsdb": 0.95, "tiff": 0.95, "toml": 0.95, "torrent": 0.95, "tsv": 0.95, "ttf": 0.95, "twig": 0.95, "txt": 0.65, "typescript": 0.95, "unknown": 0.95, "vba": 0.85, "vcxproj": 0.95, "verilog": 0.95, "vhdl": 0.95, "vtt": 0.95, "vue": 0.95, "wasm": 0.95, "wav": 0.95, "webm": 0.95, "webp": 0.95, "winregistry": 0.95, "wmf": 0.95, "woff": 0.95, "woff2": 0.95, "xar": 0.95, "xls": 0.95, "xlsb": 0.95, "xlsx": 0.95, "xml": 0.92, "xpi": 0.95, "xz": 0.95, "yaml": 0.8, "yara": 0.5, "zig": 0.95, "zip": 0.91, "zlibstream": 0.95}, "overwrite_map": {}}
```

Comment:
Note: This is not a copy of the existing file. Thresholds are being added.

Reply:
Correct! we'll need to run the various syncs, including for python. will get to it. /cc @invernizzi